### ISSUE TITLE
Default values on HomeBridge Launch

### ISSUE THREAD

**Issue by DirtyDevWork** [DirtyDevWork/homebridge-daikin#12]
Default values don't update to Daikin current values until a setting is changed and sent back to HomeBridge after HomeBridge launch

**Comment by mecrip** [comment]
HI!

I've made a quick change to get the correct init values when the homebridge server starts, by adding a call to this.getCurrentHeatingCoolingState(function (){}); at the end of the constructor.

this way the initial state is correct before any other call and homekit display the correct value for the status.
I can confirm that it works with a 12 yo  ftxs35c internal unit


thanks for the big work here!

**Comment by greensouth** [comment]
Yep, can confirm it works. Waiting for the developer to commit ;)

**Comment by extric99** [comment]
@mecrip Since this has not been committed in over 15 months is there any way to apply this fix ourselves?
